src/data.js

```diff
@@ -4,25 +4,41 @@ const data = [
     name: 'Canon EOS 5D Mark IV',
     price: '1597',
     image: '/images/Canon eos 5d mark iv.png',
+    bookmarked: false,
+    Description: 'blablabl',
```

Comment:
Maybe more meaningful data could be helpful to have a better understanding of how much space the description will need in your app.